plugins/search-react/src/components/index.ts

```diff
@@ -15,3 +15,9 @@
  */
 
 export * from './HighlightedSearchResultText';
+export * from './SearchFilter';
+export * from './SearchResult';
+export * from './SearchResultPager';
+export * from './SearchBar';
+export * from './SearchTracker';
```

Comment:
Good to remove `SearchTracker` from here, too.

Reply:
Are you sure? Do we not have for example custom components created by others that would like to wrap their component with `TrackSearch` to get analytics tracking?